snapcraft/internal/errors.py

```diff
@@ -685,3 +685,26 @@ def __init__(self, snap):
 
 class ProjectNotFoundError(SnapcraftReportableError):
     fmt = "Failed to find project files."
+
+
+class XAttributeTooLongError(SnapcraftReportableException):
```

Comment:
this should probably be a reportable error

Reply:
It is SnapcraftReportableException...? Did you mean it shouldn't be?

Reply:
oh, sorry, yes, this is good as is!